<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TemperatureDialogClass</class>
 <widget class="QDialog" name="TemperatureDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Temperature Control</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Temperature to set:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Cooler</string>
   </property>
   <widget class="QRadioButton" name="radioButtonOn" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>ON</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOff" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>80</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>OFF</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTempToSet" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>50</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>20</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderTempToSet" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>80</y>
     <width>21</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSet" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurTemp" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>210</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTempMsg" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Current Temperature:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderTempToSet</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelTempToSet</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TemperatureDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>184</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
